lib/routes/user/add_funds/add_funds_page.dart

```diff
@@ -43,71 +56,106 @@ class AddFundsState extends State<AddFundsPage> {
         _accountSubscription.cancel();
       }
     });
+    _settingsSubscription = widget._accountBloc.accountSettingsStream.listen((settings) {
+      isIpAllowed(settings.moonpayIpCheck);
+    });
+    _moonPaySubscription = _addFundsBloc.moonPayOrderStream.listen((order) {
```

Comment:
I don't think we need a subscription here, just use the stream in the build method.

Reply:
[Removed in](https://github.com/breez/breezmobile/pull/162/commits/e575a05e69ee9d79c65ba65bc54d6f1b73380a1c)